compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -462,7 +462,12 @@ class ROCMTargetBackend final : public TargetBackend {
         opt.UnsafeFPMath = false;
         opt.NoInfsFPMath = false;
         opt.NoNaNsFPMath = true;
+        // Be extra cautious while this is less tested, and prevent unknown
+        // fallbacks from global isel.
```

Comment:
Could you briefly explain what this option does and how we can observe the difference?

Reply:
I've hopefully clarified the behavior with the elaborated comment. Please let me know what you think.